static-build/cmake/Package.cmake

```diff
@@ -0,0 +1,120 @@
+set(CPACK_DEB_COMPONENT_INSTALL ON)
+
+install(DIRECTORY tarantool-prefix/bin/ DESTINATION bin
+        USE_SOURCE_PERMISSIONS
+        COMPONENT tarantool
+        EXCLUDE_FROM_ALL)
+
+install(DIRECTORY tarantool-prefix/include/ DESTINATION include
+        USE_SOURCE_PERMISSIONS
+        COMPONENT devel
+        EXCLUDE_FROM_ALL)
+
+install(DIRECTORY tarantool-prefix/bin/ DESTINATION bin
+        USE_SOURCE_PERMISSIONS
+        COMPONENT slim
+        EXCLUDE_FROM_ALL
+        FILES_MATCHING PATTERN "tarantool")
+
+set(CPACK_GENERATOR "DEB;RPM")
+set(CPACK_PACKAGE_NAME "tarantool")
+set(CPACK_PACKAGE_VERSION  $ENV{VERSION})
+set(CPACK_PACKAGE_RELEASE 1)
+set(CPACK_PACKAGE_CONTACT "tarantool@tarantool.io")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://www.tarantool.io")
+set(CPACK_PACKAGE_DIRECTORY $ENV{PACKAGE_DIR})
+set(CPACK_RPM_PACKAGE_LICENSE "BSD")
+
+set(TARANTOOL_DESCRIPTION "In-memory database with a Lua application server
+Tarantool is an in-memory database and a Lua application server.
+Its key properties include:
+.
+ * flexible data model
+ * multiple index types: HASH, TREE, BITSET
+ * optional persistence and strong data durability
+ * log streaming replication
+ * Lua functions, procedures, triggers, with rich access to database API,
+   JSON support, inter-procedure and network communication libraries
+.
+This package provides Tarantool command line interpreter and server.
+")
+
+set(TARANTOOL_DEVEL_DESCRIPTION "Tarantool is a high performance in-memory NoSQL database and Lua
+application server. Tarantool supports replication, online backup and
+stored procedures in Lua.
+
+This package provides server development files needed to create
+C and Lua/C modules.
+")
+
+set(CPACK_RPM_TARANTOOL_FILE_NAME tarantool-garbage.rpm)
+set(CPACK_DEBIAN_TARANTOOL_FILE_NAME tarantool-garbage.deb)
+
+set(CPACK_RPM_SLIM_PACKAGE_NAME "tarantool")
+set(CPACK_RPM_SLIM_PACKAGE_GROUP "Applications/Databases")
+set(CPACK_RPM_SLIM_PACKAGE_SUMMARY "In-memory database and Lua application server")
+set(CPACK_RPM_SLIM_FILE_NAME tarantool_${CPACK_PACKAGE_VERSION}.${CPACK_RPM_PACKAGE_ARCHITECTURE}.rpm)
+
+set(CPACK_RPM_DEVEL_PACKAGE_NAME "tarantool-devel")
+set(CPACK_RPM_DEVEL_PACKAGE_GROUP "Applications/Databases")
+set(CPACK_RPM_DEVEL_PACKAGE_SUMMARY "Server development files for tarantool")
+set(CPACK_RPM_DEVEL_FILE_NAME tarantool-devel_${CPACK_PACKAGE_VERSION}.${CPACK_RPM_PACKAGE_ARCHITECTURE}.rpm)
+
+set(CPACK_DEBIAN_SLIM_PACKAGE_NAME "tarantool")
+set(CPACK_DEBIAN_SLIM_PACKAGE_SECTION "database")
+set(CPACK_DEBIAN_SLIM_FILE_NAME tarantool_${CPACK_PACKAGE_VERSION}_${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}.deb)
+
+set(CPACK_DEBIAN_DEVEL_PACKAGE_NAME "tarantool-dev")
+set(CPACK_DEBIAN_DEVEL_PACKAGE_SECTION "libdevel")
+set(CPACK_DEBIAN_DEVEL_FILE_NAME tarantool-dev_${CPACK_PACKAGE_VERSION}_${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}.deb)
+
+include(CPack)
+
+cpack_add_component(tarantool
+        DISPLAY_NAME  "Binary tarantool with artefacts"
```

Comment:
Please remove one space between `DISPLAY_NAME` and `"Binary tarantool with artefacts"`.